instagram/src/Components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import './CommentStyle.css';
+
+
+class CommentSection extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            newComment: '',
+            comments: []
+          };
+    }
+
+    componentDidMount() {
+        this.setState({comments: this.props.comments });
+      }
+
+      handleAddComment = e => {
+        this.setState({ [e.target.name] : e.target.value });
+      }
+    
```

Comment:
isn't this method handle the changes in input text? Giving a declarative name to a method helps other programmers understand your code.